lists2safebrowsing.py

```diff
@@ -500,6 +544,32 @@ def main():
 
     publish_to_cloud(config, chunknum)
 
+    shavar_prod_lists_branch = {
+      'name': '70',
+      'commit': {
+        'sha': '8ffa3c6d702002e9a357dd0042848c9418e0ef8a',
+        'url': 'https://api.github.com/repos/mozilla-services/shavar-prod-lists/commits/8ffa3c6d702002e9a357dd0042848c9418e0ef8a'
+      },
+      'protected': False
+    }
+    shavar_prod_lists_non_versioning_branch = {
+      'name': 'non-versioning-branch',
+      'commit': {
+        'sha': '8ffa3c6d702002e9a357dd0042848c9418e0ef8a',
+        'url': 'https://api.github.com/repos/mozilla-services/shavar-prod-lists/commits/8ffa3c6d702002e9a357dd0042848c9418e0ef8a'
+      },
+      'protected': False
+    }
+    # get shavar_prod_lists branches from git
+    shavar_prod_lists_branches = [shavar_prod_lists_branch, shavar_prod_lists_non_versioning_branch]
+    for branch in shavar_prod_lists_branches:
+        branch_name = branch.get('name')
+        try:
+            int(branch_name)
```

Comment:
Using this to check for (numeric) version branches might pose a problem if/when we need to make a list version for, e.g., Firefox 71.0.1 ?

Reply:
I see. I didn't quite think about that since the [pref](https://searchfox.org/mozilla-central/source/modules/libpref/init/all.js#4769) only pass the major version. I can use `float` instead. Do we want the folder names to also be `70.0` instead of `70`? I wonder if we should convert the `.` to `-`... Wdyt?

Reply:
Looks like Firefox sends, e.g., https://shavar.services.mozilla.com/downloads?client=navclient-auto-ffox&appver=69.0&pver=2.2

We can make it send the full version number including point release if we update the prefs `all.js` lines to:
```
pref("browser.safebrowsing.provider.mozilla.updateURL", "https://shavar.services.mozilla.com/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2");
pref("browser.safebrowsing.provider.mozilla.gethashURL", "https://shavar.services.mozilla.com/gethash?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2");
```

Reply:
Prefer to use `.` if possible, unless that will cause path or s3 issues. In that case, we can convert to `-` characters.

Reply:
According to the "Object Key Naming Guidelines" documentation [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html) using `.` should be fine. The only down side, if it is one, is we "can’t download the object using the Amazon S3 console". 
Using. `.` should be good.

Reply:
@groovecoder, can you clarify if the branch name is `70` whether or not we should use `70.0` in the bucket name like `tracking/70.0/content-fingerprinting-track-digest256`?

Reply:
It would be great to be as consistent as possible across the Firefox version, branch name, file name, s3 key, etc.

So maybe we make the branch name `70.0` too?